docs/how-to-setup-mobile-app-locally.md

```diff
@@ -0,0 +1,140 @@
+# Setup freeCodeCamp's mobile app locally
+
+## System requirements
+
+These are the different requirements needed to contribute to the mobile application for the different operating systems. These specifications are based of running a Flutter project and two installed emulators.
+
+#### Windows
```

Comment:
We should not be stepping from `h2` to `h4`. For the most part, I suggest just emboldening these `**Windows**`

```suggestion
### Windows
```